react/src/review.jsx

```diff
@@ -4,7 +4,12 @@ import ReviewsList from './reviewsList.jsx';
 const Review = (props) => {
   return (
     <tr>
-      <td>{props.review.restaurantReviews}</td>
+      <td>
+        {props.reviewEntry.restaurantReviews.forEach(review => {
```

Comment:
Wasn't sure if this was meant to be a placeholder for more things you wanted to handle in the `.forEach()`.  Maybe a `.map()` might be better if you're trying to transform the array?  If you're planning to do more in the forEach, disregard this.